testing/tests/DevExpress.ui.widgets/speedDialAction.tests.js

```diff
@@ -339,4 +338,25 @@ QUnit.module("check action buttons click args", (hooks) => {
     });
 });
 
+QUnit.module("add visible option", (hooks) => {
```

Comment:
I want you to make test with one SDA.
Simple scenario:
1) Create, 2) set `visible` to` false`, 3) set `visible` to `true`
It is interesting because in this case we completely dispose and remove the SpeedDialMainItem

Reply:
Add test and fix